Core/Entities/Characters/enemy/EnemyTypes/Walker.cs

```diff
@@ -28,4 +28,13 @@ public Walker(Texture2D texture, Dictionary<string, Animation> animations, Motor
         ActivationRectangle.Inflate(35, 35);
         Motor.SetEnemy(this);
     }
+    
+    public Walker(Level level, Dictionary<string, Animation> animations) : base(Vector2.Zero, ContentLoader.EnemyTexture, new RandomMotor(level), level)
```

Comment:
Wenn wir davon ausgehen dass jeder Walker immer ein Blob ist, könnte man die Animation auch im Konstruktor festlegen ohne die Animation zu übergeben.

Reply:
Kann gerne Mal schauen, ob ich das mit dem Shot nochmal anpassen kann. Denke das könnte mit dem ground-depth layer anstatt nur ground gehen. 
Und wegen den enemies können wir ja gerne Mal gemeinsam in der Gruppe überlegen, was welcher enemy sein soll, dann kann das natürlich ggf weg :)